docs/items/items.md

```diff
@@ -30,4 +30,27 @@ Registering an Item
 
 Items must be [registered][registering] to function.
 
-[registering]: ../concepts/registries.md#registering-things
+Coloring an Item
+----------------
+
+There are items, textures of which are colored within the code, and in fact, there are a lot of such items. For example, leather helmets, spawn eggs, potions and other.
+
+### Item Color Handlers
+
+An item color handler is required to color an item and is an instance of `IItemColor`, and therefore implements the following method.
+
+```java
+int getColorFromItemstack(
+  ItemStack stack, 
+  int tintIndex)
+```
+
+This method returns a hex representation of a color in an integer.
+
+Tint indexes are specified for faces of an element in item's model JSON file. When a face with a tint index is being drawn, its tint index is passed to the method. A face without a tint index won't be colored. Layer indexes are used as tint indexes for items, models of which inherit `builtin/generted` model.
```

Comment:
"When a face with a tint index is being drawn, its tint index is passed to the method. A face without a tint index won't be colored."

The first sentence is not necessary here. This could just be "A face without a tint index won't be colored, and therefore will not have its color handler called." or something